app/src/pages/project/SpansTable.tsx

```diff
@@ -72,10 +73,66 @@ const PAGE_SIZE = 50;
 
 type RootSpanFilterValue = "root" | "all";
 
+const defaultColumnSettings = {
```

Comment:
I only added this to the Spans table specifically to prevent the metadata col from getting too small.